stories/settings.stories.js

```diff
@@ -19,10 +19,138 @@ import TabBar from '@material/react-tab-bar';
  */
 import { __ } from '@wordpress/i18n';
 
+const setupSettings = () => {
+	window.googlesitekit.modules.analytics[ 'get-accounts' ] = {
```

Comment:
This should be set in `googlesitekit.admin.datacache`, with the actual cache key that would be generated by Site Kit.

Reply:
I tried this approach and had difficulty getting the set values to stick.